src/Microsoft.DotNet.Interactive.Tests/LanguageServices/HoverTextTests.cs

```diff
@@ -116,6 +117,36 @@ public async Task language_service_methods_run_deferred_commands(Language langua
                 .ContainEquivalentOf(new FormattedValue(expectedMimeType, expectedContent));
         }
 
+        [Theory]
+        [InlineData(Language.CSharp, "var s = new SampleLibrary.Sample$$Library();")]
+        [InlineData(Language.FSharp, "let s = SampleLibrary.Sample$$Library()")]
+        public async Task hover_text_can_read_doc_comments_from_xml_files(Language language, string markupCode)
```

Comment:
Does this approach give us any assurances over:

* Docs for BCL types?
* Docs for types coming from NuGet packages?

Reply:
For (1), no; my initial PR comment tried to explain this.  The runtime assemblies don't have a .xml file next to them, so that'll be a separate issue to solve.  For (2), it depends on if the NuGet package _has_ the xml files.  If it does, then it'll get picked up, if not there's nothing we can do.

Reply:
Ah, sorry. I missed that comment and just dove right into the code in stereotypical engineer fashion. Sounds great.

Reply:
I'm working on some tests for NuGet packages that I'll add to the PR.